README.md

```diff
@@ -5,7 +5,14 @@ This is a set of common stuff which might be helpfull with your day to day EpiSe
 ## Installation
 
 It's available as a [nuget package](https://www.nuget.org/packages/Forte.EpiCommonUtils/)
- 
+
+Then, in your Startup you need to register package by running
```

Comment:
Then, in your Startup, you need to register the package by running
I just put it in Grammarly :)